include/open62541/client.h

```diff
@@ -280,6 +284,28 @@ UA_Client_connectUsername(UA_Client *client, const char *endpointUrl,
     /* Silence a false-positive deprecated warning */
     return UA_Client_connect(client, endpointUrl);
 }
+#ifdef UA_ENABLE_ENCRYPTION
+/* Connect to the server and create+activate a Session with the given X.509 certificate.
+ * This first set the X509IdentityToken in the client config and
+ * then calls the regular connect method. */
+static UA_INLINE UA_StatusCode
+UA_Client_connectCertificate(UA_Client *client, const char *endpointUrl,
+                             UA_ByteString certificate, UA_ByteString privateKey) {
+    UA_X509IdentityToken* identityToken = UA_X509IdentityToken_new();
```

Comment:
`UA_Client_connectCertificate` is incomplete.
It does nothing with the private key.

Maybe we cannot make this work without a hard link to some specific plugin.
But we will not change the rule that the core library has no dependency on specific plugins.

In that case we could just document how the `authSecurityPolicies` field is to be set up before calling the normal _connect.
And reference `UA_ClientConfig_setCertAuthentication` in a comment.